### PR TITLE
Refactor OAuth Mediator properties

### PR DESCRIPTION
**Pull request by dilee**
## Purpose
Refactoring OAuth Mediator properties.

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java [hunk 1]
```diff
@@ -123,13 +126,12 @@ public Composite createFigure(final Composite parent, final FormToolkit widgetFa
...
 		CompositionSequence oAuthMediatorStep = new BindingCompositionSequence(propertiesEditionComponent);
 		CompositionStep propertiesStep = oAuthMediatorStep.addStep(EsbViewsRepository.OAuthMediator.Properties.class);
-		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.description);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.commentsList);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.reverse);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.remoteServiceUrl);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.username);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.password);
-		
+        propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.description);
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java]: Correct formatting.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java [hunk 2]
```diff
@@ -397,10 +399,11 @@ public void keyPressed(KeyEvent e) {
 
 	
 	protected Composite createUsernameText(FormToolkit widgetFactory, Composite parent) {
-		createDescription(parent, EsbViewsRepository.OAuthMediator.Properties.username, EsbMessages.OAuthMediatorPropertiesEditionPart_UsernameLabel);
-		username = widgetFactory.createText(parent, ""); //$NON-NLS-1$
+	    filterCredentialsSubPropertiesGroup = EEFPropertyViewUtil.createSubsectionGroup(widgetFactory, parent, "Credentials", true);
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java]: Correct formatting.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java [hunk 3]
```diff
@@ -456,7 +459,7 @@ public void keyPressed(KeyEvent e) {
 		});
 		EditingUtils.setID(username, EsbViewsRepository.OAuthMediator.Properties.username);
 		EditingUtils.setEEFtype(username, "eef::Text"); //$NON-NLS-1$
-		FormUtils.createHelpButton(widgetFactory, parent, propertiesEditionComponent.getHelpContent(EsbViewsRepository.OAuthMediator.Properties.username, EsbViewsRepository.FORM_KIND), null); //$NON-NLS-1$
+		FormUtils.createHelpButton(widgetFactory, filterCredentialsSubPropertiesGroup, propertiesEditionComponent.getHelpContent(EsbViewsRepository.OAuthMediator.Properties.username, EsbViewsRepository.FORM_KIND), null); //$NON-NLS-1$
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java]: Limit line length to 120 characters.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java (hunk 1 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/806

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java (hunk 2 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/806

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java (hunk 3 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/806

**Pull request merged**